index.js

```diff
@@ -160,8 +161,8 @@ SocketIoEngine.prototype.step = function (requestSpec, ee) {
       let delegateFunc = self.httpDelegate.step(requestSpec, ee);
       return delegateFunc(context, callback);
     }
-    ee.emit('counter', 'engine.socketio.emit', 1);
-    ee.emit('rate', 'engine.socketio.emit_rate');
+    ee.emit('counter', 'socketio.emit', 1);
```

Comment:
what is the rationale for this change? is this a broader artillery 2.0 change or?

Reply:
Yes, I've adapted the way metrics are tracked from artillery 2.0 (see https://github.com/artilleryio/artillery/blob/main/packages/core/lib/engine_socketio.js) so they will show up properly in reports.